src/App.js

```diff
@@ -9,14 +9,7 @@ import Footer from "./components/Footer";
 // Pages
 import Main from "./pages/Main";
 import Comments from "./pages/Comments";
-import SearchPage from "./pages/Search";
-import MoviesTab from "./pages/MoviesTab";
-import TvShowsTab from "./pages/TvShowsTab";
-import DevTeam from "./pages/DevTeam";
-import ShowtimesTab from "./pages/ShowtimesTab";
-import Dashboard from "./pages/Dashboard";
-import PodCast from "./pages/Podcast";
-
```

Comment:
I think we might need these imports at the app level

Reply:
i didnt delete these idk whats up with this